core/object/object.h

```diff
@@ -411,11 +413,20 @@ public:
 		return String(#m_class);                                                                                                                 \
 	}                                                                                                                                            \
 	virtual const StringName *_get_class_namev() const override {                                                                                \
-		static StringName _class_name_static;                                                                                                    \
-		if (unlikely(!_class_name_static)) {                                                                                                     \
-			StringName::assign_static_unique_class_name(&_class_name_static, #m_class);                                                          \
+		static std::queue<StringName> _class_name_statics;                                                                                       \
```

Comment:
STL containers are not allowed, see https://docs.godotengine.org/en/stable/contributing/development/cpp_usage_guidelines.html#standard-template-library